meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/u-boot/u-boot-xlnx/uz/lmp-base.cfg

```diff
@@ -10,7 +10,7 @@ CONFIG_RSA=y
 CONFIG_ENV_IS_IN_FAT=y
 # CONFIG_ENV_IS_IN_SPI_FLASH is not set
 CONFIG_ENV_FAT_INTERFACE="mmc"
-CONFIG_ENV_FAT_DEVICE_AND_PART="0:1"
+CONFIG_ENV_FAT_DEVICE_AND_PART="1:1"
```

Comment:
I don't think this is correct.  `uz` represents all of the UltraZed devices which actually do have eMMC.

Reply:
I understand but `lmp.cfg` and `lmp-ebbr.cfg` for uz already configured to use SD for uboot env:

```
./uz/lmp-base.cfg:CONFIG_ENV_FAT_DEVICE_AND_PART="0:1"
./uz/lmp-ebbr.cfg:CONFIG_ENV_FAT_DEVICE_AND_PART="1:1"
./uz/lmp.cfg:CONFIG_ENV_FAT_DEVICE_AND_PART="1:1"
./k26/lmp-base.cfg:CONFIG_ENV_FAT_DEVICE_AND_PART="0:1"
./k26/lmp-ebbr.cfg:CONFIG_ENV_FAT_DEVICE_AND_PART="1:1"
./k26/lmp.cfg:CONFIG_ENV_FAT_DEVICE_AND_PART="1:1"

```

Reply:
Ugh, we'll need to figure out what our "out of the box" experience is for this HW.  Do we expect users to use an SD card or flash the eMMC?

Reply:
@igoropaniuk I withdraw my comment here.  Looking at the latest documentation, it is expected that this HW will be booted from SD card.

Reply:
@igoropaniuk could we add a reference to the documentation to avoid confusion in the future?

Reply:
@MrCry0 unfortunately, it's not published yet :( (it expected to be deployed to docs.foundries.io after v89 release)

Reply:
@igoropaniuk, just something which prevents confusion again :)|

Like, "Due to the latest documentation, it is expected that this HW will be booted from an SD card."

Reply:
done